### PR TITLE
docs: Clarification when working with model pagination.

### PR DESCRIPTION
**Pull request by iRedds**
**Description**
Topics related to pagination periodically appear on the forum. Some developers do not understand how it works.
Therefore, I want to suggest a clarification in the documentation for the Pagination library when using the ``Model::paginate()`` method

**Checklist:**
- [x] Securely signed commits
- [ ] Component(s) with PHPDoc blocks, only if necessary or adds value
- [ ] Unit testing, with >80% coverage
- [x] User guide updated
- [ ] Conforms to style guide

### CODE REVIEW

File: user_guide_src/source/libraries/pagination.rst [hunk 1]
```diff
@@ -56,6 +56,33 @@ The first element is the results from the database, **users**, which is retrieve
 the Model will hold on to the instance it used and store it in the public class variable, **$pager**. So, we grab
 that and assign it to the $pager variable in the view.
 
+.. important:: It is important to understand that the Model::paginate() method uses the Model and QueryBuilder methods.
+Therefore, trying to use ``$db->query()`` and Model::paginate() **will not work** because ``$db->query()`` executes
+the query immediately and is not associated with a QueryBuilder.
```
Comment (paulbalandan) [inline user_guide_src/source/libraries/pagination.rst:61]: You need to indent once these two lines.

Comment (iRedds) [reply user_guide_src/source/libraries/pagination.rst]: Thank you. Missed it.

File: user_guide_src/source/libraries/pagination.rst [hunk 2]
```diff
@@ -56,6 +56,33 @@ The first element is the results from the database, **users**, which is retrieve
 the Model will hold on to the instance it used and store it in the public class variable, **$pager**. So, we grab
 that and assign it to the $pager variable in the view.
 
+.. important:: It is important to understand that the Model::paginate() method uses the Model and QueryBuilder methods.
+    Therefore, trying to use ``$db->query()`` and Model::paginate() **will not work** because ``$db->query()`` executes
+    the query immediately and is not associated with a QueryBuilder.
+
+To define conditions for pagination in a model, you can::
+
+    //You can specify conditions directly.
```
Comment (kenjis) [inline user_guide_src/source/libraries/pagination.rst:65]:
```suggestion
    // You can specify conditions directly.
```

**Review by kenjis — APPROVED** [review]
This is a great contribution.

**Review by paulbalandan — APPROVED** [review]

**Pull request merged**